bridge/server/utils/curl.utils.ts

```diff
@@ -1,28 +1,5 @@
-import { WebhookConfig } from '../../shared/models/webhook-config';
-
-function generateWebhookConfigCurl(webhookConfig: WebhookConfig): string {
-  let params = '';
-  for (const header of webhookConfig?.header || []) {
-    params += `--header '${header.name}: ${header.value}' `;
-  }
-  params += `--request ${webhookConfig.method} `;
-  if (webhookConfig.proxy) {
-    params += `--proxy ${webhookConfig.proxy} `;
-  }
-  if (webhookConfig.payload) {
-    let stringify = webhookConfig.payload;
-    try {
-      stringify = JSON.stringify(JSON.parse(webhookConfig.payload));
-    } catch {
-      stringify = stringify.replace(/\r\n|\n|\r/gm, '');
-    }
-    params += `--data '${stringify}' `;
-  }
-  return `curl ${params}${webhookConfig.url}`;
-}
-
-function parseCurl(curl: string): { [key: string]: string[] } {
-  const startCommand = 'curl ';
+function parseCurl(curl: string, ignoreStart = false): { [key: string]: string[] } {
```

Comment:
Can curl be null

Reply:
No. The interface/webhook-version that is used here provides `string[]` for parsing the curl commands. It's either an empty array or a string.